python/grass/pygrass/vector/basic.py

```diff
@@ -134,7 +134,7 @@ def contains(self, point):
         """
         return bool(
             libvect.Vect_point_in_box(
-                point.x, point.y, point.z if point.z else 0, self.c_bbox
+                point.x, point.y, point.z or 0, self.c_bbox
```

Comment:
Not sure about  this one, as only points.z is tested...
There are more of this cases below...

Reply:
That I'm pretty sure its not problematic. This code wasn't unpacking a tuple, it was calling `libvect.Vect_point_in_box()` with 4 arguments. The pattern I see here, is that if the point coordinates doesn't have a 3rd dimension (z), it sets z to 0 in order to call the function that checks if a point is in a box. I'm pretty sure I have seen that pattern in mathematical formulations of spatial algorithms outside of GRASS